tools/MUXPGODatabase/restore-pgodb.ps1

```diff
@@ -1,47 +1,89 @@
+Param(
+    [Parameter(Position=0, Mandatory=$true)]
+    [string] $NuGetConfigPath
+)
+
 . .\version.ps1
 . .\template.ps1
 . .\config.ps1
 
-$feedUri = "https://pkgs.dev.azure.com/ms/microsoft-ui-xaml/_packaging/MUX-Dependencies/nuget/v2"
+function Get-AvailablePackages ( $package )
+{
+    $result = @()
+
+    $output = ( & nuget.exe list $package -prerelease -allversions -configfile $NuGetConfigPath )
 
-$currentVersion = MakeVersion $releaseVersionMajor $releaseVersionMinor ( GetDatetimeStamp $pgoBranch )
+    if ( $LastExitCode -ne 0 )
+    {
+        throw "FAILED: nuget.exe list"
+    }
 
-Write-Host ( "PGO OPTIMIZE: requesting {0} version {1}" -f $packageId, ( FormatVersion $currentVersion ) )
+    foreach ( $line in $output )
+    {
+        $name, $version = $line.Split(" ")
 
-$packageSource = Register-PackageSource -ForceBootstrap -Name MUX_Dependencies -Location $feedUri -ProviderName NuGet -Trusted
-$packages = ( Find-Package $packageId -Source MUX_Dependencies -AllowPrereleaseVersions -AllVersions ) | Sort-Object -Property Version -Descending
+        # Some packages on the feed use an old versioning scheme. Skip the package if it does not include a -foo portion in the version.
+        if($version.Contains("-"))
```

Comment:
Why is this necessary?  This should be taken care of when version obj is created.

Reply:
It was tripping up on some package versions with versions like "2.0.0" - I'm not sure where those came from - probably from when PGO was first getting set up.

Reply:
Good idea. I've delisted the packages and removed this condition.